### ISSUE TITLE
Switch to Tauri?

### ISSUE THREAD

**Issue by intercepted16** [Levminer/screentime#11]
It appears you already have some Tauri apps, with similar functionality, such as minimizing to system tray. Tauri would reduce the bundle size hugely, and also greatly improve performance. Why not switch?

**Comment by Levminer** [comment]
Yes I wanted to migrate Screentime to Tauri but I couldn't find a Rust equivalent to Electron's [powerMonitor ](https://www.electronjs.org/docs/latest/api/power-monitor). If you know an alternative let me know
